### ISSUE TITLE
NaN SSD Loss

### ISSUE THREAD

**Issue by hnushrat** [Socret360/object-detection-in-keras#20]
While training, loss is nan on the custom dataset.

Here is the error which is displayed:
1/562 [..............................] - ETA: 1:39:05 - loss: nanBatch 0: Invalid loss, terminating training
/home/user/Shared Tasks/proj/Self_driving_cars/object-detection-in-keras/utils/bbox_utils/iou.py:31: RuntimeWarning: divide by zero encountered in true_divide
  res = intersect / union
562/562 [==============================] - 51s 73ms/step - loss: nan - val_loss: nan
